runtime/mainnet/src/lib.rs

```diff
@@ -600,6 +601,7 @@ impl Contains<RuntimeCall> for SafeModeWhitelistedCalls {
 			RuntimeCall::System(_)
 			| RuntimeCall::SafeMode(_)
 			| RuntimeCall::TxPause(_)
+			// balance transfers are allowed
 			| RuntimeCall::Balances(_) => true,
```

Comment:
not sure why this is permitted even in the template, if in safe mode, we want to allow transfers? that seems to defy the whole purpose of safe mode.

Reply:
good point, let's remove it

Reply:
Same comment about:

```
/// Calls that cannot be paused by the tx-pause pallet.
pub struct TxPauseWhitelistedCalls;
/// Whitelist `Balances::transfer_keep_alive`, all others are pauseable.
impl Contains<RuntimeCallNameOf<Runtime>> for TxPauseWhitelistedCalls {
	fn contains(full_name: &RuntimeCallNameOf<Runtime>) -> bool {
		match (full_name.0.as_slice(), full_name.1.as_slice()) {
			(b"Balances", b"transfer_keep_alive") => true,
			_ => false,
		}
	}
}
```